draft-irtf-cfrg-aegis-aead.md

```diff
@@ -1677,7 +1677,8 @@ AEGIS has been shown to have reforgeability resilience in {{FLLW17}}. Without th
 
 AEGIS-128X and AEGIS-256X share the same security properties and requirements as AEGIS-128L and AEGIS-256 respectively. In particular, the security level and usage limits remain the same {{D23}}.
 
-AEGIS is considered secure against guess-and-determine attacks aimed at recovering the state from observed ciphertexts. This resilience extends to quantum adversaries in the Q1 model, wherein quantum attacks do not confer any practical advantage for decrypting previously recorded ciphertexts or achieving key recovery.
+AEGIS is considered secure against guess-and-determine attacks aimed at recovering the state from observed ciphertexts.
+This resilience extends to quantum adversaries operating within the Q1 model, where the attacker has access to a quantum computer but is restricted to classical (non-quantum) communications with the systems under attack. In this model, quantum attacks offer no practical advantage in decrypting previously recorded ciphertexts or in recovering the encryption key.
```

Comment:
Is this meant to be on a new line? Otherwise, LGTM.

Reply:
This is just for clarity in the Markdown file. In generated files, there will be no line breaks.

Reply:
Do you suggest  there should be a new line? The paragraph is a bit long. OTOH, this is all directly related.

Reply:
> Do you suggest there should be a new line? The paragraph is a bit long. OTOH, this is all directly related.

I think you could get away with it either way.

Reply:
Let's add a new line, then, if only for the text version of the document.